src/stores/TxStore.js

```diff
@@ -41,8 +41,9 @@ class TxStore {
           this.getTxReceipt(hash)
         }).on('error', (e) => {
           if(!e.message.includes('not mined within 50 blocks') && !e.message.includes('Failed to subscribe to new newBlockHeaders')){
-            this.alertStore.setLoading(false)
-            this.alertStore.pushError('Transaction rejected on wallet');
+            // False error is thrown by web3 https://github.com/ethereum/web3.js/issues/2542
```

Comment:
@akolotov An issue from web3 it throwing error `Error: Transaction has been reverted by the EVM:` even if transaction was successful. The only workaround I found until it is fixed in a new version was to comment the code to stop the loading flow of the app when a transaction is submitted. The bad side is that if transaction is really reverted, the UI won't be notified